docs/development/setup-advanced.md

```diff
@@ -67,8 +67,10 @@ git remote add -f upstream https://github.com/zulip/zulip.git
 sudo apt-get install closure-compiler libfreetype6-dev libffi-dev \
     memcached rabbitmq-server libldap2-dev redis-server \
     postgresql-server-dev-all libmemcached-dev python3-dev \
-    python-dev python-virtualenv hunspell-en-us git yui-compressor \
-    puppet gettext postgresql
+    python-dev python-virtualenv hunspell-en-us git \
+    yui-compressor puppet gettext postgresql \
+    libssl-dev libxml2-dev libxslt1-dev libjpeg8-dev zlib1g-dev \
+    libcurl4-openssl-dev
```

Comment:
This last one doesn't exist on Trusty.

Reply:
Oh!! how did you find out?
As far as I know, `libcur14-openssl-dev` has problems installing due it's uncommon dependency `librtmp-dev`. We can include it too here and things should be good to go.

Reply:
Apparently I didn't check correctly, and it actually does exist.  But what I attempted was `apt-cache search libcurl4-openssl-dev`.

Reply:
Nice! So, it's ready to merge?